src/main/java/frc/robot/Controls.java

```diff
@@ -101,11 +101,20 @@ private void configureDrivebaseBindings() {
                   ChassisSpeeds targetSpeeds =
                       new ChassisSpeeds(
                           -driverController.getLeftY()
-                              * MaxSpeed, // Drive forward with negative Y (forward)
+                              * MaxSpeed
+                              * (driverController.start().getAsBoolean()
+                                  ? 0.5
+                                  : 1), // Drive forward with negative Y (forward)
                           -driverController.getLeftX()
-                              * MaxSpeed, // Drive left with negative X (left)
+                              * MaxSpeed
+                              * (driverController.start().getAsBoolean()
+                                  ? 0.5
+                                  : 1), // Drive left with negative X (left)
```

Comment:
Along with, or instead of simply modifying the maximum value, consider squaring or cubing the driverController output so the same stick movement results in smaller resulting values. If squaring, be sure to separately add in the sign of the original output value. If cubing, no need for that.

Reply:
This is a trick I've seen used to let the drive controller still have full range of speed / rotation, while letting small stick motions be more precise and large stick movements be more granular.